tests/test_cli_system_health_coverage.py

```
@@ -0,0 +1,87 @@
+from pathlib import Path
+from unittest.mock import MagicMock, patch
+
+sys.path.insert(0, os.path.join(os.path.dirname(__file__), "..", "src"))
+
+from mmrelay.cli import _print_system_health
+
+
+class TestPrintSystemHealthDatabase(unittest.TestCase):
+    """Tests for _print_system_health database journal mode check."""
+
+    def _make_paths_info(self, db_path):
```

Ruff (0.15.10)

[warning] Missing return type annotation for private function `_make_paths_info`

(ANN202)